composeApp/src/desktopMain/kotlin/org/ooni/probe/Main.kt

```diff
@@ -21,16 +21,27 @@ import ooniprobe.composeapp.generated.resources.app_name
 import ooniprobe.composeapp.generated.resources.tray_icon
 import org.jetbrains.compose.resources.painterResource
 import org.jetbrains.compose.resources.stringResource
+import org.ooni.probe.data.DeepLinkHandler
 import org.ooni.probe.data.models.DeepLink
+import org.ooni.probe.shared.DesktopOS
+import org.ooni.probe.shared.Platform
 import java.awt.Desktop
 
-fun main() {
-    val autoLaunch = AutoLaunch(appPackageName = "org.openobservatory.ooniprobe")
+val APP_ID = "org.openobservatory.ooniprobe"
+
+fun main(args: Array<String>) {
+    val autoLaunch = AutoLaunch(appPackageName = APP_ID)
```

Comment:
Is it ok to run `AutoLaunch` for every instance started? Or should this be after the Unique4j call?

Reply:
`Unique4j` only applies to `windows` and `linux`  we should allow `AutoLaunch`  as it is for now. No problem encountered the way it is set up.